src/main/java/seedu/duke/Parser.java

```diff
@@ -11,6 +15,8 @@
  */
 public class Parser {
 
+    private ItemContainer items;
```

Comment:
Since all command in parser is static, and we are only interfacing with one ItemClass in v1.0, it will be easier if we just instantiate one static ItemClass